### ISSUE TITLE
[BUG] Crash on loading

### ISSUE THREAD

**Issue by itsdorkney** [FaeWulf/Diversity#20]
### Current Behavior

just installed the latest version and im getting an error then crash when trying to load the game up using neoforge

![image](https://github.com/user-attachments/assets/dfcbd02e-7fc9-4800-a8b7-ce05a79e01ae)

### Expected Behavior

for the game to load lol

### Additional info

```markdown
- **Game version**: 1.21.1
- **Mod version**: 2.1.1 (neoforge)
```

### Crash log/ Gamelog

[crash-2024-10-29_14.56.41-fml.txt](https://github.com/user-attachments/files/17558926/crash-2024-10-29_14.56.41-fml.txt)

**Comment by FaeWulf** [comment]
Thanks for the report! I will update the fixed version tomorrow!

**Comment by FaeWulf** [comment]
Issue should be fixed in version 2.1.2, tysm!
https://modrinth.com/mod/faewufs-diversity/versions
